src/config/subgraph.ts

```diff
@@ -20,7 +20,8 @@ const SUBGRAPH_URLS = {
 
   [AVALANCHE]: {
     stats: "https://subgraph.satsuma-prod.com/3b2ced13c8d9/gmx/gmx-avalanche-stats/api",
-    referrals: "https://subgraph.satsuma-prod.com/3b2ced13c8d9/gmx/gmx-avalanche-referrals/api",
+    referrals:
+      "https://subgraph.satsuma-prod.com/3b2ced13c8d9/gmx/gmx-avalanche-referrals/version/master-240415215829-f6877d6/api",
```

Comment:
is there some specific change only on this version?
why only avalanche? let's deploy it first and change url back here?